llvm/test/CodeGen/RISCV/rvv/vl-opt-instrs.ll

```diff
@@ -5069,3 +5069,63 @@ define <vscale x 4 x float> @vfwmaccbf16_vf(<vscale x 4 x float> %a, bfloat %b,
   %2 = call <vscale x 4 x float> @llvm.riscv.vfadd(<vscale x 4 x float> poison, <vscale x 4 x float> %1, <vscale x 4 x float> %d, iXLen 7, iXLen %vl)
   ret <vscale x 4 x float> %2
 }
+
+define <vscale x 4 x half> @vfsqrt(<vscale x 4 x half> %a) {
```

Comment:
Do we need +zvfh or +zvfbfmin? Can we just test it with floats + fadd e.g. how `@vfwmaccbf16_vf` does it